python/src/trezorlib/transport/webusb.py

```diff
@@ -75,7 +75,7 @@ def close(self) -> None:
                 self.handle.releaseInterface(self.interface)
                 self.handle.close()
             except Exception as e:
-                raise TransportException(f"USB close failed: {e}") from e
+                raise CloseFailed(f"USB close failed: {e}") from e
```

Comment:
wondering if we even need to propagate the exception. can you see a reason for it?

Reply:
Only to let the user know there was an USB error (in case the device stopped responding not because of the `RebootToBootloader` request)... so probably emitting a warning log will suffice.
WDYT?

Reply:
agreed :+1:

Reply:
https://github.com/trezor/trezor-firmware/pull/5000/commits/2d354cf7b9c431f421e306400fcf9461e6396635